bw2analyzer/lci.py

```diff
@@ -1,14 +1,60 @@
 import pandas as pd
-from bw2data import get_activity
+from bw2data import Database
 from bw2calc import LCA
+from typing import List, Union
 
 
-def get_labeled_inventory(lca: LCA) -> pd.DataFrame:
+def _get_dependent_database_names(lca: LCA) -> List[str]:
```

Comment:
Duplicates existing functionality, and assumes presence of `remapping_dicts`. Please use [Database.find_dependents](https://github.com/brightway-lca/brightway2-data/blob/8627519fd9a01cc6504b10278f7948a627e58b63/bw2data/backends/base.py#L135) or [Data.find_graph_dependents](https://github.com/brightway-lca/brightway2-data/blob/8627519fd9a01cc6504b10278f7948a627e58b63/bw2data/backends/base.py#L161) with the functional unit. I know this is a bit more work, but it is worth it.

Reply:
You can merge if you just want to get this done with, but at some point we should eliminate duplicate functionality, it will cause pain in the future when things change.

Reply:
No worries, I will eliminate it before merging. What is your opinion on the metadata caching (see #15)?